syntax = "proto3";

service FileService {
    rpc getFile(GetFile) returns (FileResult) {}
}

message GetFile {
    string file_name = 1;
}

message FileResult {
    string file_name = 1;
    int32 total_parts = 2;
    repeated Parts parts = 3;
}

message Parts {
    bytes data = 1;
}